### PR TITLE
Hw17

### PR DESCRIPTION
**Pull request by andrjuxa201185**

### CODE REVIEW

File: src/components/show/Show.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import './show.scss';
+
+export class Show extends Component {
+  state = {
+    btnText: 'Show',
+    show: ''
+  }
```
Comment (iliatcymbal) [inline src/components/show/Show.js]: It is better to have true/false flag in state `state = { isActive: false }` and change it `this.setState({ isActive: !this.state.isActive })`
And the in jsx 
```
<button
  className={this.state.isActive ? 'active' : ''}
  onClick={this.showContent}
>
  {this.state.isActive ? 'Hide' : 'Show'}
</button>
...
```

File: .eslintrc.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "react/sort-comp": 0,
     "react/prop-types": 0,
     "react/no-array-index-key": "off",
+    "react/destructuring-assignment": 0,
```
Comment (iliatcymbal) [inline .eslintrc.json]: I wouldn't switch off this rule. Anyway, I will check it manually and ask to use destructuring where you don't.

File: src/components/tasklist/Tasklist.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  render() {
+    return (
+      <div className="tasklist">
+        <h3>Hello, {this.state.userName}</h3>
+        <p>You have {this.state.tasks} tasks</p>
+        <p>Done: {this.state.doneTasks}</p>
+        <p>In progress: {this.state.progressTasks}</p>
+        <p>Waiting: {this.state.waitingTasks}</p>
```
Comment (iliatcymbal) [inline src/components/tasklist/Tasklist.js]: use destructuring for `this.state`.

**Comment by andrjuxa201185** [comment]
@iliatcymbal,  Is it better?

**Pull request merged**